### PR TITLE
Startup error messaging

### PR DESCRIPTION
**Pull request by MikeV-TalentConsulting**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -177,6 +177,10 @@
                                 {
                                     "name": "EnvironmentName",
                                     "value": "[parameters('environmentName')]"
+                                },
+								{
```
Comment (HussainsGitHub) [inline azure/template.json:181]: Could the indentation be fixed for the ResourceEnvironmentName app settings please

File: azure/template.json [hunk 2]
```diff
@@ -333,6 +337,10 @@
...
```
Comment (HussainsGitHub) [inline azure/template.json:341]: and here too

File: azure/template.json (hunk 1 above)
Comment (tom-gough) [reply azure/template.json]: Sorted.

File: azure/template.json (hunk 2 above)
Comment (tom-gough) [reply azure/template.json]: Sorted.

**Review by HussainsGitHub — APPROVED** [review]

**Pull request merged**